subjects/podiumposition/README.md

```diff
@@ -0,0 +1,41 @@
+## podiumposition
+
+### Instructions
+
+A F1 race just finished and the commentator is calling the finishing positions incorrectly.
+Help to fix this before the contestants arrive at the podium by providing the commentator with the correct podium position.
+
+Write a function `PodiumPosition` that takes an array of arrays of type `string` and returns the competitor positions correctly.
+
+### Expected function
+
+```go
+func PodiumPosition(podium[][] string) [][]string {
```

Comment:
The space for the parameter is in the wrong place

Reply:
It is fixed.